File: c_language/Sejong/일반_C/인증/212차모의인증/3.c
#include <stdio.h>

int main(){
    int x[50], i ,j, num, n[50], count[50], tmp, c = 0, check, ecount;
    scanf("%d", &num);
    for(i = 0; i < num; i++){
        check = 0;
        scanf("%d", &tmp);
        x[i] = tmp;
        for(j = 0; j < i; j++){
            if(x[i] == x[j]){
                check = 1;
                break;
            }
        }
        if(check != 1){
            n[c] = tmp;
            c++;
        }
    }
    for(i = 0; i < c; i++){
        ecount = 0;
        for(j = 0; j < num; j++){
            if(n[i] == x[j]){
                ecount++;
            }
        }
        count[i] = ecount;
    }

    for(i = 0; i < c; i++){
        printf("%d %d\n", n[i], count[i]);
    }
     

    return 0;
}

File: Python/baekjoon/func/3.py
import sys

def judgement(n):
	num = [int(i) for i in str(n)]
	if len(num) <= 2:
		return 1
	diff = num[0] - num[1]
	for i in range(1, len(num) - 1):
		check = num[i] - num[i + 1]
		if diff != check:
			return -1
	return 1

n = int(sys.stdin.readline())
count = 0
	
for i in range(1, n + 1):
	if judgement(i) == 1:
		count += 1
# judgement(210)
print(count)
